development/states/custom-rpc.json

```diff
@@ -158,7 +158,7 @@
       }
     ],
     "selectedAddress": "0xfdea65c8e26263f6d9a1b5de9555d2931a33b825",
-    "network": "loading",
+    "network": "166",
```

Comment:
I put it in its own commit so it has a note in the git history.

This state file should represent a custom RPC being selected, but this attribute had the state also representing a loading state.

I set it to a hard network ID so it doesn't show loading.

I came up with 166 because at the time I thought that was the consensysnet ID, but actually it's 161.